src/Infrastructure/Services/BindleService.cs

```diff
@@ -13,4 +13,12 @@ public BindleService(IConfiguration configuration)
         var bindleUrl = configuration.GetValue<string>("Bindle:Url", "http://127.0.0.1:8080/v1");
         _client = new BindleClient(bindleUrl);
     }
+
+    public async Task<List<string>> GetBindleRevisionNumbers(string bindleId)
+    {
+        return (await _client.QueryInvoices(bindleId))
+            .Where(i => i.Bindle.Name == bindleId) // this should be replaced with a direct API call when available
```

Comment:
Nu prea stiu de ce avem acest comentariu aici. Practic vad ca bindleId se trimite si catre metoda query, doar ca ea nu iti garanteaza ca o sa ai doar invoice-uri cu nume identice cu bindleId-ul, deci acest where pare ca nu e ceva temporar.

Reply:
Aici temporar se refera pana apare un endpoint in bindle prin care sa tragem direct invoice-ul dupa id